src/custom/constants/tokens/index.ts

```diff
@@ -137,3 +137,18 @@ export const ADDRESS_IMAGE_OVERRIDE = {
   [V_COW_TOKEN_MAINNET.address]: vCowLogo,
   [COW_TOKEN_MAINNET.address]: cowLogo,
 }
+
+/**
+ * Locked GNO COW claiming
+ */
+export const MERKLE_DROP_CONTRACT_ADDRESSES: Record<number, string> = {
```

Comment:
what are these changes? are you sure about this?

Reply:
ok make sure to comment that or put it in the pr description otherwise idk what ur doing. could be wrong addresses for example

Reply:
Updated with a comment.